docs/Getting Started/Alpine Linux/Root on ZFS/2-system-installation.rst

```diff
@@ -40,23 +40,18 @@ System Installation
 
 #. Create boot pool::
 
-     tee -a /root/grub2 <<EOF
-     # Features which are supported by GRUB2
-     async_destroy
-     bookmarks
-     embedded_data
-     empty_bpobj
-     enabled_txg
-     extensible_dataset
-     filesystem_limits
-     hole_birth
-     large_blocks
-     lz4_compress
-     spacemap_histogram
-     EOF
-
-      zpool create \
-          -o compatibility=/root/grub2 \
+      zpool create -d \
+          -o feature@async_destroy=enabled \
```

Comment:
What do you think about using https://github.com/openzfs/zfs/pull/11818 ?

Reply:
It is not possible to use compatibility= option on Alpine, because Alpine does not package the necessary files [source](https://pkgs.alpinelinux.org/contents?branch=edge&name=zfs&arch=aarch64&repo=main).